test/scenarios/mappings/all_features.json

```diff
@@ -0,0 +1,50 @@
+[
+  {
```

Comment:
this one mapping doesn't have category check - so why did `id:2` from input got filtered out ?
shouldn't it be in output with just `{ discount: 10 } ` ?